<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticleEvaluationToolClass</class>
 <widget class="QMainWindow" name="ParticleEvaluationToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Daten</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="DataTableView"/>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="CalcualteResultsButton">
          <property name="text">
           <string>Auswerten</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ResultTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Auswertung</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableView" name="ResultsTable">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="ActionImportData"/>
    <addaction name="ActionExportData"/>
    <addaction name="separator"/>
    <addaction name="ActionSaveResults"/>
    <addaction name="separator"/>
    <addaction name="ActionSaveProject"/>
    <addaction name="ActionOpenProject"/>
    <addaction name="separator"/>
    <addaction name="ActioClose"/>
   </widget>
   <widget class="QMenu" name="ResultsMenu">
    <property name="title">
     <string>Auswerten</string>
    </property>
    <addaction name="ActionRestartCalculation"/>
    <addaction name="ActionContiniuslyCalculation"/>
    <addaction name="separator"/>
    <addaction name="actionCSV_Export"/>
   </widget>
   <widget class="QMenu" name="SettingsMenu">
    <property name="title">
     <string>Einstellungen</string>
    </property>
    <addaction name="ActionResultSettings"/>
    <addaction name="ActionProgrammSettings"/>
   </widget>
   <widget class="QMenu" name="HelpMenu">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="LicenseAction"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="ResultsMenu"/>
   <addaction name="SettingsMenu"/>
   <addaction name="HelpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ActionImportData">
   <property name="text">
    <string>Daten Importieren</string>
   </property>
  </action>
  <action name="ActionExportData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Daten Exportieren</string>
   </property>
  </action>
  <action name="ActionSaveResults">
   <property name="text">
    <string>Auswertung Speichern</string>
   </property>
  </action>
  <action name="ActionSaveProject">
   <property name="text">
    <string>Projekt Speichern</string>
   </property>
  </action>
  <action name="ActionOpenProject">
   <property name="text">
    <string>Projekt Öffnen</string>
   </property>
  </action>
  <action name="ActioClose">
   <property name="text">
    <string>Schließen</string>
   </property>
  </action>
  <action name="LicenseAction">
   <property name="text">
    <string>Lizens</string>
   </property>
  </action>
  <action name="ActionResultSettings">
   <property name="text">
    <string>Auswertungseinstellungen</string>
   </property>
  </action>
  <action name="ActionProgrammSettings">
   <property name="text">
    <string>Programmeinstellungen</string>
   </property>
  </action>
  <action name="ActionRestartCalculation">
   <property name="text">
    <string>Auswertung Starten</string>
   </property>
  </action>
  <action name="ActionContiniuslyCalculation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dauerhaft Auswerten</string>
   </property>
  </action>
  <action name="actionCSV_Export">
   <property name="text">
    <string>CSV Export</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
